test/llvm/with-typed-ptrs/lifetime-intrinsics/lifetime-multi-blocks.chpl

```diff
@@ -20,9 +20,6 @@ proc mytest_multi_blocks() {
   // CHECK: %test1_chpl = alloca double
   // CHECK-NEXT: %[[REG5:[0-9]+]] = bitcast double* %test1_chpl to i8*
   // CHECK-NEXT: call void @llvm.lifetime.start.p0i8(i64 8, i8* %[[REG5]])
-  // CHECK: %{{[0-9]+}} = bitcast double* %a_chpl to i8*
```

Comment:
What's the reason for removing these? Are these instructions no longer present, and if so why?

Reply:
They were present only because they were input for `llvm.invariant.start`. The are irrelevant for the purpose of these llvm.lifetime.start / llvm.lifetime.end tests.